tests/test_sample_rayservice_yamls.py

```diff
@@ -62,7 +62,14 @@ def wait(self):
                 namespace = self.namespace, label_selector='ray.io/node-type=worker')
             serve_services = k8s_v1_api.list_namespaced_service(
                 namespace = self.namespace, label_selector =
-                f"ray.io/originated-from=RayService_{self.custom_resource_object['metadata']['name']}-serve")
```

Comment:
This is a bug why #1830 fails on the master branch. See [this build](https://buildkite.com/ray-project/ray-ecosystem-ci-kuberay-ci/builds/3043#018d37a3-6e72-4856-b2f7-ec4710ce6046) for more details.

Reply:
Oh, I mistakenly treated the `ray.io/serve` as the `ray.io/service`.